app/src/main/java/fr/free/nrw/commons/di/ApplicationlessInjection.java

```diff
@@ -0,0 +1,99 @@
+package fr.free.nrw.commons.di;
+
+import android.app.Activity;
+import android.app.Service;
+import android.content.BroadcastReceiver;
+import android.content.ContentProvider;
+import android.content.Context;
+import android.support.v4.app.Fragment;
+
+import javax.inject.Inject;
+
+import dagger.android.AndroidInjector;
+import dagger.android.DispatchingAndroidInjector;
+import dagger.android.HasActivityInjector;
+import dagger.android.HasBroadcastReceiverInjector;
+import dagger.android.HasContentProviderInjector;
+import dagger.android.HasFragmentInjector;
+import dagger.android.HasServiceInjector;
+import dagger.android.support.HasSupportFragmentInjector;
+
+public class ApplicationlessInjection
+        implements
+        HasActivityInjector,
+        HasFragmentInjector,
+        HasSupportFragmentInjector,
+        HasServiceInjector,
+        HasBroadcastReceiverInjector,
+        HasContentProviderInjector {
+
+    private static ApplicationlessInjection instance = null;
+
+    @Inject
+    DispatchingAndroidInjector<Activity> activityInjector;
+    @Inject
+    DispatchingAndroidInjector<BroadcastReceiver> broadcastReceiverInjector;
+    @Inject
+    DispatchingAndroidInjector<android.app.Fragment> fragmentInjector;
+    @Inject
+    DispatchingAndroidInjector<Fragment> supportFragmentInjector;
+    @Inject
+    DispatchingAndroidInjector<Service> serviceInjector;
+    @Inject
+    DispatchingAndroidInjector<ContentProvider> contentProviderInjector;
```

Comment:
Line wrapping - many other places in the app put the annotations on the same line.

Reply:
Done.